backend/coins/eth/db/db.go

```diff
@@ -91,9 +91,11 @@ func (tx *Tx) PutOutgoingTransaction(transaction *types.TransactionWithMetadata)
 
 type byNonce []*types.TransactionWithMetadata
 
-func (txs byNonce) Len() int           { return len(txs) }
-func (txs byNonce) Less(i, j int) bool { return txs[i].Transaction.Nonce() < txs[j].Transaction.Nonce() }
-func (txs byNonce) Swap(i, j int)      { txs[i], txs[j] = txs[j], txs[i] }
+func (txs byNonce) Len() int      { return len(txs) }
```

Comment:
what happened here? why dfferent order, and why reformatted?

Reply:
that's more recent goimports, which newever golangci-lint uses, too.
`go get -u golang.org/x/tools/cmd/goimports` and try formatting. you should get the same result.

re: different order. it just looked nicer. I can revert but `Less` method would still be formated on multiline.